source/process/sg_rest_markup.rst

```diff
@@ -86,23 +96,13 @@ Create numbered lists and procedure steps using arabic numerals for the top-leve
 Links to external web pages
 ---------------------------
 
-The quickest and easiest way to create a link in reStructuredText is to use the URL without any extra markup. For example:
-
-.. code-block:: none
-  
-  https://www.mattermost.org/manifesto/
-
-On output to HTML, the link looks like this: https://www.mattermost.org/manifesto/.
-
-You can also create a link that has link text. For example:
```

Comment:
By removing this, are you trying to say that "naked" links like this https://www.mattermost.org/manifesto/ are not allowed, and that all links must have a name and be of the form `` `Mattermost Manifesto <https://www.mattermost.org/manifesto/>`_``?